### ISSUE TITLE
Getting a blank window when executing some code

### ISSUE THREAD

**Issue by eladcandroid** [lukehaas/runjs#5]
Here's my code:

```javascript

async function f() {
  console.log('check');
  let promise = new Promise((resolve, reject) => {
    setTimeout(() => resolve("done!"), 1000)
  });

  let result = await promise; // wait till the promise resolves (*)

  alert(result); // "done!"
}

f();

```

After I close the blank window, the code isn't running anymore so I have to restart the app..

![screen shot 2018-12-19 at 11 43 10](https://user-images.githubusercontent.com/13297000/50212426-bdb70c80-0383-11e9-838a-b981b9f96ab2.png)

**Comment by lukehaas** [comment]
Oh, this is very interesting. Definitely a bug. Thanks for raising this.

**Comment by lukehaas** [comment]
This is now resolved in version 1.1.0
